full/Dockerfile

```
@@ -1,7 +1,8 @@
 FROM buildpack-deps:disco
 
 ### base ###
-RUN yes | unminimize \
+RUN apt-get update -yqq \
```

On a side note, I think you don't need `-y` on `apt-get update` (there is no confirmation prompt).

```suggestion
RUN apt-get update -qq \
```